benchmark/script.py

```diff
@@ -168,6 +169,34 @@ def get_config_by_name(section, name):
     return None
 
 
+def get_cluster(options, env):
+    """
+    Obtain a cluster from the command line options and environment.
+
+    :param BenchmarkOption options: Parsed command line options.
+    :param dict env: Dictionary of environment variables.
+    :return BenchmarkCluster: Cluster to benchmark.
+    """
+    cluster_option = options['cluster']
+    if cluster_option:
+        try:
+            cluster = BenchmarkCluster.from_cluster_yaml(
+                FilePath(cluster_option)
+            )
+        except IOError as e:
+            usage(
+                options, 'Cluster file {!r} not found.'.format(e.filename)
+            )
+    else:
+        try:
+            cluster = BenchmarkCluster.from_acceptance_test_env(env)
+        except KeyError as e:
+            usage(
+                options, 'Environment variable {!r} not set.'.format(e.args[0])
+            )
```

Comment:
This could also fail if the JSON env var isn't valid JSON, and maybe if the other things have invalid input. Should probably handle those cases, since good error messages are one of the best ways for users to learn how to do things. (Citation needed).